src/App.scss

```diff
@@ -332,6 +332,9 @@ nav.usa-breadcrumb {
 .usa-table--striped tbody tr:nth-child(even) td,
 .usa-table--striped tbody tr:nth-child(even) th {
   background-color: $gray-light;
+  .usa-checkbox {
+    background-color: $gray-light;
+  }
```

Comment:
No, these are our updates to the USDWS style guide that we use, and are things we want to fix on all of our pages. 

If you are seeing a .usa-* class added to the wrong element, then please block that PR as it is going to cause bugs and require unneeded overrides. The only reason we should be overriding .usa-* classes is because our HCD team does not like the default behavior provided by our style guide.